syntax = "proto3";

package visitcounter;

service VisitService {
  rpc GetVisitCount(Request) returns (VisitResponse);
  rpc HealthCheck(Request) returns (HealthCheckResponse);
}


message Request {}


message VisitResponse {
  int64 visit_count = 1;
  int64 local_visit_count = 2;
  string host = 3;
}


message HealthCheckResponse {
  string version = 1;
  string timestamp = 2;
}
